examples/table/table.html

```diff
@@ -42,7 +42,7 @@ <h2 id="ex_label">Example</h2>
     <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
     <div id="ex1">
       <div role="table" aria-label="Students" aria-describedby="students_table_desc">
```

Comment:
According to our [table naming guidance](https://w3c.github.io/aria-practices/#naming_with_captions), the aria-describedby is superfluous on table elements if a caption is present. I assume this would be true if the the element is an ARIA caption as well.